assets/src/GameBoard.ts

```diff
@@ -77,28 +76,33 @@ function within<T>(ctx: Ctx, fn: ((...x: any[]) => T)): T {
 }
 
 function drawSnakeBody(layer: Ctx, snake: Snake, prepare?: Function) {
-  const points = P.shrink(P.smooth(snake.coords), 0.12);
+  const points = shrink(snake.coords, 0.62);
```

Comment:
The fix here was two-fold: 
1) remove the `shrink` call because I'm not even sure it's needed...
2) modify the scale value because of removing the `shrink` call